lib/logstash/inputs/neo4j.rb

```diff
@@ -0,0 +1,87 @@
+# encoding: utf-8
+require "logstash/inputs/base"
+require "logstash/namespace"
+require "logstash/inputs/neo4j-client"
```

Comment:
We can remove this require since we already do a lazy require in the register method.